# execute goal
geometry_msgs/PoseStamped target_pose
---
bool success
string message